<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnmpGetWidget</class>
 <widget class="QWidget" name="SnmpGetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>125</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SNMP GET</string>
  </property>
  <property name="toolTip">
   <string>SNMP GET</string>
  </property>
  <property name="statusTip">
   <string>SNMP GET</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0">
     <property name="horizontalSpacing">
      <number>4</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="lblSnmpProfile">
       <property name="text">
        <string>SNMP profile</string>
       </property>
       <property name="buddy">
        <cstring>cmbSnmpProfile</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="lblTimeout">
       <property name="text">
        <string>Timeout</string>
       </property>
       <property name="buddy">
        <cstring>spinTimeout</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="btnSnmpCredentials">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../qHostMonitor.qrc">
         <normaloff>:/img/browse.png</normaloff>:/img/browse.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblHostPort">
       <property name="text">
        <string>Host[:port]</string>
       </property>
       <property name="buddy">
        <cstring>cmbHostPort</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="cmbHostPort">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>localhost</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cmbSnmpProfile">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>public</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QSpinBox" name="spinTimeout">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QSpinBox" name="spinRetries">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="lblRetries">
       <property name="text">
        <string>Retries</string>
       </property>
       <property name="buddy">
        <cstring>spinRetries</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="Bevel6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="lblOid">
       <property name="text">
        <string>Alert when OID</string>
       </property>
       <property name="buddy">
        <cstring>cmbOid</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbOid">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnMibBrowser">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../qHostMonitor.qrc">
         <normaloff>:/img/browse.png</normaloff>:/img/browse.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QComboBox" name="cmbCriteria">
       <item>
        <property name="text">
         <string>is &lt; than</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>is &gt; than</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>is = to</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>is &lt;&gt; from</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>contains</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>doesn't contain</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>increases by ..</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>decreases by ..</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>changes by ..</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>increases by ..(%)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>decreases by ..(%)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>changes by ..(%)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>increases /sec</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>decreases /sec</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>changes /sec</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbValue">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGetValue">
       <property name="toolTip">
        <string>Get current value</string>
       </property>
       <property name="statusTip">
        <string>Get current value</string>
       </property>
       <property name="text">
        <string>Get value</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
